x-pack/test/security_solution_endpoint_api_int/config.ts

```diff
@@ -29,6 +29,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         // always install Endpoint package by default when Fleet sets up
         `--xpack.fleet.packages.0.name=endpoint`,
         `--xpack.fleet.packages.0.version=latest`,
+        `--xpack.fleet.enableExperimental.0=diagnosticFileUploadEnabled`, // this will be removed in 8.7 when the feature is released
```

Comment:
Could you also add this to our non-api test FTR suite here: 

https://github.com/elastic/kibana/blob/6561ba86d216aaf73d9b968f749bf1a034514cd5/x-pack/test/security_solution_endpoint/config.ts#L43-L55

Reply:
Done 👍